docker/mcp/services.py

```
@@ -0,0 +1,85 @@
+        raw_content = soup.get_text(separator=' ').strip()
+        
+        # Process content
+        content = mcp_process_content(raw_content)
+        
+        return SearchResult(
+            title=result.get('title', 'Untitled'),
+            url=url,
+            snippet=result.get('snippet', ''),
+            content=content
+        )
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)